fuzz/Cargo.toml

```diff
@@ -0,0 +1,90 @@
+[package]
+name = "bitcoin-fuzz"
+edition = "2018"
+version = "0.0.1"
+authors = ["Generated by fuzz/generate-files.sh"]
+publish = false
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+honggfuzz = { version = "0.5", default-features = false }
+bitcoin = { version = "0.30.0", features = [ "serde" ] }
+
+rust-crypto = "0.2"
```

Comment:
We don't need this anymore, right?

Reply:
Correct. I've fixed it locally but haven't pushed, sorry.